app/helpers/link_helper.rb

```diff
@@ -17,6 +17,6 @@ def govuk_styled_link(text, path: nil, inverse: false)
   end
 
   def govuk_styled_links_list(links)
```

Comment:
Ah, I think I was assuming that this would be an all or nothing thing - that you'd have:

`def govuk_styled_links_list(links, inverse: false)`

...is there a market for having some links in a single list inverse, and some not?

Reply:
Hmm, that's a good shout. Will update.